Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+
+Definitions of this type work for event class and object `extends` definitions both for the normal case (subclass / subtype) as well as the "patch extends" case, however "hidden" event classes and objects are not supported due to the potential of colliding observable `type_id` values. See [Appendix 2: Hidden Types](#appendix-2-hidden-types) for more about this issue.
+
+Example in object definition file `objects/cve.json`:
+```jsonc
+{
+  "name": "cve",
+  // ... other object definition fields
+  "attributes": {
+    // ... other attributes
+    "uid": {
+      "caption": "CVE ID",
+      // ... other attribute fields
+
+      // This defines uid as an observable when directly inside a cve object
+      "observable": 18
+    },
+    // ...
+  },
+  // ...
+}
+```
+
+Event classes work identically: attribute observable definitions occur in attribute detail objects inside the `attributes` object.
+
+### Definition Example: Observable by Class-Specific Attribute Path
+This last observable definition type allows defining an observable for an attribute inside a nested structure for an event class, though it can be used for attributes directly defined in the class (in this case being a alternative to defining event class attribute observables). This type of definition is done by adding an `observables` object to the event class definition that maps from attribute paths to observable `type_id` values.
+
+Class-specific attribute path definitions also work for event class `extends` definitions both for the normal case (subclass / subtype) as well as the "patch extends" case. In the extends cases, the class-specific observable definitions replace a prior definition or add a new definition. 
+
+Hidden event classes are not supported due to the potential of colliding observable `type_id` values. See [Appendix 2: Hidden Types](#appendix-2-hidden-types) for more about this issue. 
+
+The attribute paths use a simple dotted notation. See [Appendix 1: Attribute Paths](#appendix-1-attribute-paths) for details about how attribute paths are defined in OCSF. Notably these attribute paths do _not_ contain array references. In this context it means all array items in the attribute path are considered.
+
+The following is an example showing the definition pattern for observables by class-specific attributes in an example "Bag" event class that holds an array of items, where each item has a `type_id` field that we want to become observables.
+
+Example in event class definition file `events/bag.json`:
+```jsonc
+{
+  "name": "bag",
+  // ... other event class definition fields
+  "attributes": {
+    // ... other attributes
+    
+    // Here was add the items array, where each item has (at least) a type_id attribute
```

Comment:
Grammar of the comment: "Here was add the items array...".  Here we add?

Reply:
The comment was confusing and added unnecessary detail. I simplified it:
```jsonc
  // Here we add the items array: an array of item objects
```